app/services/distribution_itemized_breakdown_service.rb

```diff
@@ -64,16 +61,12 @@ def distributions
   def current_onhand_quantities(inventory)
     if inventory
       inventory.all_items.group_by(&:name).to_h { |k, v| [k, v.sum(&:quantity)] }
-    else
-      organization.inventory_items.group("items.name").sum(:quantity)
     end
   end
 
   def current_onhand_minimums(inventory)
     if inventory
```

Comment:
@dorner I think this condition is no longer needed -- OK to blow up if inventory is nil now